pocs/observatory.py

```diff
@@ -342,23 +342,49 @@ def update_tracking(self):
         if self.current_offset_info is not None:
             self.logger.debug("Updating the tracking")
 
+            # Get the pier side of pointing image
```

Comment:
Could you add some more comments? I'm certainly ignorant about what is happening here, and the lack of problem description in the PR isn't helping.